src/test/java/com/artipie/YamlPermissionsTest.java

```diff
@@ -48,10 +53,11 @@ void johnCanDownloadDeployAndDelete() throws Exception {
             this.permissions(),
             new AllOf<YamlPermissions>(
                 new ListOf<org.hamcrest.Matcher<? super YamlPermissions>>(
-                    new MatcherOf<>(perm -> { return perm.allowed(uname, "delete"); }),
-                    new MatcherOf<>(perm -> { return perm.allowed(uname, "deploy"); }),
-                    new MatcherOf<>(perm -> { return perm.allowed(uname, "download"); }),
-                    new MatcherOf<>(perm -> !perm.allowed(uname, "install"))
+                    // @checkstyle LineLengthCheck (4 lines)
+                    new MatcherOf<>(perm -> { return perm.allowed(new Authentication.User(uname), "delete"); }),
+                    new MatcherOf<>(perm -> { return perm.allowed(new Authentication.User(uname), "deploy"); }),
+                    new MatcherOf<>(perm -> { return perm.allowed(new Authentication.User(uname), "download"); }),
+                    new MatcherOf<>(perm -> !perm.allowed(new Authentication.User(uname), "install"))
```

Comment:
@olenagerasimova I'd suggest to extract variable for `Authentication.User(uname)` to make code easier to read here and in `janeCanDownloadAndDeploy`, `adminCanDoAnything` methods